sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -282,19 +292,21 @@ result = [doc for doc in response if not doc.is_error]
 for doc in result:
     for entity in doc.entities:
         print("Entity: {}".format(entity.name))
-        print("Url: {}".format(entity.url))
+        print("URL: {}".format(entity.url))
         print("Data Source: {}".format(entity.data_source))
         for match in entity.matches:
             print("Score: {0:.3f}".format(match.score))
             print("Offset: {}".format(match.offset))
             print("Length: {}\n".format(match.length))
 ```
 
+The returned response is a heterogeneous list of result and error objects: list[[RecognizeLinkedEntitiesResult](https://aka.ms/azsdk-python-textanalytics-recognizelinkedentitiesresult), [DocumentError](https://aka.ms/azsdk-python-textanalytics-documenterror)]
+
 Please refer to the service documentation for a conceptual discussion of [entity linking](https://docs.microsoft.com/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-entity-linking)
 and [supported types](https://docs.microsoft.com/azure/cognitive-services/text-analytics/named-entity-types?tabs=general).
 
 ### Extract key phrases
-Extract key phrases in a batch of documents.
+Extract key phrases in text to determine the main talking points.
```

Comment:
NIT: It will be great to add an example, like you did in the LinkedEntities section